app/components/AnalystDashboard/AllDashboard.tsx

```diff
@@ -185,8 +186,9 @@ const AllDashboardTable: React.FC<Props> = ({ query }) => {
   const [columnFilters, setColumnFilters] = useState<MRT_ColumnFiltersState>(
     []
   );
+  const showLeadFetureFlag = useFeature('show_lead').value ?? false;
   const [columnVisibility, setColumnVisibility] = useState<MRT_VisibilityState>(
-    {}
+    { Lead: showLeadFetureFlag }
```

Comment:
I think this is causing an error, since we're instantiating the state with the growthbook flag that isn't evaluated until it reaches the client side so there's a mismatch (at least when I set the growthbook flag to false).

It also reverts back to the lead being shown on page load